dataactcore/models/jobModels.py

```diff
@@ -35,8 +49,10 @@ class Submission(Base):
     datetime_utc = Column(DateTime)
     user_id = Column(Integer, nullable=False) # This refers to the users table in the User DB
     cgac_code = Column(Text)
-    reporting_start_date = Column(Date)
-    reporting_end_date = Column(Date)
+    reporting_start_date = Column(Date, nullable=False)
+    reporting_end_date = Column(Date, nullable=False)
+    reporting_fiscal_year = Column(Integer, nullable=False, default=generateFiscalYear, server_default='0')
```

Comment:
Thanks for the clarification! Then my question is...why do we need the `server_default` here? We would never want these fields to be set to 0, so having both options in the model was confusing when I read this.

We so something similar when loading the `tas` field on the staging tables, and there is no `server_default` on those models. If there's a good reason to have it, I'll make a note to go change that so we're consistent.